content/learn.md

```diff
@@ -247,6 +247,10 @@ dasharo_faq:
 
         If you don’t have a solution yet, let Dasharo become your answer. [Contact us](/#get-a-quote)
 
+    - title: What kind of consulting services that 3mdeb can provide other than online course?
+      content: >-
+        In addition to training, we provide in-house development and maintenance to keep hardwar and softwar up to date, or provide ad-hoc support to your development team.
```

Comment:
content is generally not wrapped in this file, but can we start wrapping new content?

Reply:
@pietrushnic Automatic wrapping (Alt+Q) dosen't wok, We would have to do it manually... worth it?

Reply:
@MateuszKochner I have no idea what editor you use. My editor don't have any feature under Alt+Q.  Tools should not  prevent high quality content preparation. Please correctly configure tools you using.

I'm repeated argument about 80 character wrapping many times: https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings

This request is not about whim, but the way most programmers in open-source software world trained their brains.  Reviewing long lines is harder and slower.

Reply:
@pietrushnic after wrapping all sections, render was broken (the point is we are using markdown header styling, not markdown content). I don't think effort of wrapping everything in this document is needed here.

Reply:
@artur-rs it is very hard to me to do review in code that has not wrapped lines. I will not insist if this breaks wesbite, but it doesn't mean I think it is ok. Disagree and commit.